### PR TITLE
runtime_tools: Clarify scheduler utilization docs

### PR DESCRIPTION
**Pull request by sverker**
From issue #4502.

### CODE REVIEW

**Comment by sverker** [comment]
@tak30 How about this

@rickard-green What do you think about "a couple of seconds" a recommended minimum measuring interval for scheduler utilization?

**Comment by tak30** [comment]
@sverker Seems fine. That will avoid the misunderstanding I had with the output given by the utilization function and the way samples are needed to take.

File: lib/runtime_tools/doc/src/scheduler.xml [hunk 1]
```diff
@@ -117,6 +120,29 @@
         <p>Calculate scheduler utilizations for the time interval from when
 	<c><anno>Sample</anno></c> was taken and "now". The same as calling
 	<c>scheduler:utilization(Sample, scheduler:sample_all())</c>.</p>
+	<note>
+	  <p>
+	    Scheduler utilization is meassured as an average value over a time
```
Comment (josevalim) [inline lib/runtime_tools/doc/src/scheduler.xml:125]:
```suggestion
	    Scheduler utilization is measured as an average value over a time
```

**Review by rickard-green — APPROVED** [review]

File: lib/runtime_tools/doc/src/scheduler.xml [hunk 2]
```diff
@@ -63,7 +63,10 @@
 	<taglist>
 	  <tag><c>{normal, SchedulerId, Util, Percent}</c></tag>
 	  <item>Scheduler utilization of a normal scheduler with number
-	    <c>SchedulerId</c>.</item>
+	    <c>SchedulerId</c>. Even schedulers that are not online will be
```
Comment (paulo-ferraz-oliveira) [inline lib/runtime_tools/doc/src/scheduler.xml:66]:
```suggestion
	    <c>SchedulerId</c>. Schedulers that are not online will also be
```

File: lib/runtime_tools/doc/src/scheduler.xml [hunk 3]
```diff
@@ -117,6 +120,29 @@
...
+	    have passed between the two samples. For this reason, you should not
+	    do
+	  </p>
+<pre>
+scheduler:utilization(scheduler:sample()). % DO NOT DO THIS!
+</pre>
+          <p>
+            The above example takes two samples in rapid succession and calculates
+	    the scheduler utilization between them. The resulting values will
+	    probably be more misleading rather than informative.
```
Comment (paulo-ferraz-oliveira) [inline lib/runtime_tools/doc/src/scheduler.xml:137]:
```suggestion
	    probably be more misleading than informative.
```

**Comment by paulo-ferraz-oliveira** [comment]
`recon` [does it like so](https://github.com/ferd/recon/blob/master/src/recon.erl#L359). We usually call that with `5000` (ms) and it works OK.

**Pull request merged**